tensorflow_model/mnist_convnet_keras_tensorflow2.py

```diff
@@ -0,0 +1,116 @@
+# Python 3.6.0
+# tensorflow 1.1.0
```

Comment:
Ohh!! sorry I forgot to change the version name. kindly just change the version name